FabricTestExample/src/Test1864.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import { Button, View, Text } from 'react-native';
+import { NavigationContainer, ParamListBase } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import { NativeStackNavigationProp } from '../../lib/typescript/native-stack';
```

Comment:
I would change the import to `react-native-screens/native-stack`.

Reply:
Auto import 🤷🏻‍♂️ 

https://github.com/software-mansion/react-native-screens/pull/1866/commits/ab40baae5ec7d55fab398fc9acea2a42a32640c7